bgpd/bgp_vty.c

```diff
@@ -1760,10 +1760,10 @@ DEFPY (bgp_session_dscp,
        bgp_session_dscp_cmd,
        "bgp session-dscp (0-63)$dscp",
        BGP_STR
-       "Override default (C6) bgp TCP session DSCP value\n"
-       "Manually configured dscp parameter\n")
+       "Override default (CS6) DSCP for BGP connections\n"
+       "Manually configured DSCP value\n")
 {
-	bm->tcp_dscp = dscp << 2;
```

Comment:
dscp should be changed also, since it's really a tos m, where dscp is just encoded inside it.

Reply:
Ignore me, I misread the whole DEFPY here...